lib/Generate.js

```diff
@@ -52,26 +52,13 @@ function readUtf8(filepath) {
   });
 }
 
-function generateElmJson(projectRootDir/*:string*/, testRootDir/*:string*/, pathToElmBinary /*:string*/, filePaths /*:Array<string>*/) {
+function generateElmJson(projectRootDir/*:string*/, testRootDir/*:string*/, pathToElmBinary /*:string*/, filePaths /*:Array<string>*/, hasBeenGivenCustomGlobs /*int*/) {
```

Comment:
I'm assuming `/*int*/` was meant to be `/*:boolean*/` here! 😄 

```suggestion
function generateElmJson(projectRootDir/*:string*/, testRootDir/*:string*/, pathToElmBinary /*:string*/, filePaths /*:Array<string>*/, hasBeenGivenCustomGlobs /*:boolean*/) {
```

Reply:
If only I had a compiler! Good spot !